openapi/v2/device-sdk.yaml

```diff
@@ -1254,6 +1254,36 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /validate:
```

Comment:
I suggest to rename this path to `/validate/device` to prevent we have further validator requirements in the future.